h2o-algos/src/main/java/hex/glrm/GLRM.java

```diff
@@ -770,7 +809,7 @@ public void computeImpl() {
         model._output._history_step_size.add(step);
         // Transpose Y' to get original Y
         model._output._archetypes = yt.buildTable(model._output._names_expanded, false);
-        if (_parms._recover_svd) recoverSVD(model, xinfo);
+        recoverSVD(model, xinfo, tempinfo);     // getting variance information here
```

Comment:
For this JIRA, we need to recover PCA like metrics which can be done automatically in method recoverSVD.  Therefore, I removed the parameter in order to get the metrics.